components/Table/Table.js

```diff
@@ -95,12 +113,184 @@ export class Table {
     this.tableHeader.append(tableHeaderRow);
   }
 
+  renderDropDownMenu(menuBtn, isSortable, cell) {
+    const dropDownMenu = document.createElement('div');
+    dropDownMenu.classList.add('drop-down__menu');
+    this.menuDropDownItems = this.menuConfig
+      .filter((elConfig) => {
+        if (
+          this.callbacksMap.sortCallback ||
+          this.callbacksMap.filterCallback
+        ) {
+          const menuElSort =
```

Comment:
Let's call some more descriptive to denote, that this is a boolean flag, something like `isSortEl`